src/applications/disability-benefits/all-claims/pages/secondaryUploadSourcesChoice.js

```diff
@@ -4,18 +4,24 @@ import { uploadDescription } from '../content/secondaryUploadSourcesChoice';
 export const uiSchema = index => ({
   'ui:title': ptsd781aNameTitle,
   'ui:description': uploadDescription,
-  [`view:uploadChoice${index}`]: {
-    'ui:title': 'Do you have supporting documents you would like to upload?',
-    'ui:widget': 'yesNo',
+  [`secondaryIncident${index}`]: {
+    'view:uploadSources': {
+      'ui:title': 'Do you have supporting documents you would like to upload?',
+      'ui:widget': 'yesNo',
+    },
   },
 });
 
 export const schema = index => ({
   type: 'object',
   properties: {
-    [`view:uploadChoice${index}`]: {
-      type: 'boolean',
-      properties: {},
+    [`secondaryIncident${index}`]: {
```

Comment:
Cool, so now you're grouping the data for each secondary incident under the secondary incident key rather than making each piece root-level?